aspnetcore/fundamentals/aot/request-delegate-generator/diagnostics/rdg007.md

```diff
@@ -30,7 +30,7 @@ Types that are used for surrogate binding via the `AsParameters` attribute must
 
 Remove the non-public constructor, or add a new public constructor.
 
-:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/aot/diagnostics/Rdg7/Program.cs" id="snippet_1f" highlight="13-15,17-22":::
```

Comment:
Fixup to get the highlighting right:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/fad5845f-df32-4836-8b81-2fa9854ca225)